app/components/provider_interface/header_component.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="govuk-header__logo provider-header__logo">
```

Comment:
```suggestion
    <div class="govuk-header__logo app-provider-header__logo">
```

Wondering if we should be consistent in how we namespace custom classes… `app-` is typically the norm across GOV.UK services.

Reply:
We should. I've added a card to our tech debt backlog about this: https://trello.com/c/RzV6xdX8/333-review-css-classnames-preferring-app-prefixes-for-custom-css